server/repository/src/db_diesel/vaccine_course/vaccine_course_row.rs

```diff
@@ -0,0 +1,89 @@
+use super::vaccine_course_row::vaccine_course::dsl::*;
+
+use serde::Deserialize;
+use serde::Serialize;
+
+use crate::RepositoryError;
+use crate::StorageConnection;
+
+use diesel::prelude::*;
+
+table! {
+    vaccine_course (id) {
+        id -> Text,
+        name -> Text,
+        program_id -> Text,
+        demographic_indicator_id -> Text,
+        coverage_rate -> Double,
+        is_active -> Bool,
+        wastage_rate -> Double,
+        doses -> Integer,
+    }
+}
+
+#[derive(
+    Clone, Insertable, Queryable, Debug, PartialEq, AsChangeset, Serialize, Deserialize, Default,
+)]
+#[diesel(table_name = vaccine_course)]
+pub struct VaccineCourseRow {
+    pub id: String,
+    pub name: String,
+    pub program_id: String,
+    pub demographic_indicator_id: String,
+    pub coverage_rate: f64,
+    pub is_active: bool,
+    pub wastage_rate: f64,
```

Comment:
I noticed we don't have wastage rate on the front end. I wonder if we will use this elsewhere?
Maybe generic to a facility /  or facility level so we could apply different rates of wastage for the same course at different levels.

Maybe you had come to a conclusion about this elsewhere?

Reply:
Hey @fergie-nz - the wastage rates needs to be applied at the `vaccine course` level - we'll need a way for central users to add a percentage value via the UI